docs/metrics.md

```diff
@@ -1,154 +1,127 @@
 
-### kubevirt_vmi_memory_usable_bytes
-#### HELP kubevirt_vmi_memory_usable_bytes The amount of memory which can be reclaimed by balloon without causing host swapping in bytes.
-The amount of memory which can be reclaimed by balloon without causing host swapping in bytes.
+### kubevirt_vmi_outdated_count
+Indication for the number of VirtualMachineInstance workloads that are not running within the most up-to-date version of the virt-launcher environment.
 
-### kubevirt_vmi_memory_used_total_bytes
-#### HELP kubevirt_vmi_memory_used_total_bytes The amount of memory in bytes used by the domain.
-The amount of memory in bytes used by the domain.
+### kubevirt_vmi_phase_count
+Sum of VMIs per phase and node.
 
-### kubevirt_vmi_cpu_affinity
-#### HELP kubevirt_vmi_cpu_affinity vcpu affinity details
-The vcpu affinity details
+`phase` can be one of the following: [`Pending`, `Scheduling`, `Scheduled`, `Running`, `Succeeded`, `Failed`, `Unknown`]
 
 ### kubevirt_vmi_storage_flush_requests_total
-#### HELP kubevirt_vmi_storage_flush_requests_total storage flush requests.
 Storage flush requests.
 
 ### kubevirt_vmi_storage_flush_times_ms_total
-#### HELP kubevirt_vmi_storage_flush_times_ms_total total time (ms) spent on cache flushing.
 Total time (ms) spent on cache flushing.
 
 ### kubevirt_vmi_storage_iops_read_total
-#### HELP kubevirt_vmi_storage_iops_read_total I/O read operations
 I/O read operations
 
 ### kubevirt_vmi_storage_iops_write_total
-#### HELP kubevirt_vmi_storage_iops_write_total I/O write operations
 I/O write operations
 
 ### kubevirt_vmi_storage_read_times_ms_total
-#### HELP kubevirt_vmi_storage_read_times_ms_total Storage read operation time
 Storage read operation time
 
 ### kubevirt_vmi_storage_read_traffic_bytes_total
-#### HELP kubevirt_vmi_storage_read_traffic_bytes_total Storage read traffic in bytes
 Storage read traffic in bytes
 
 ### kubevirt_vmi_storage_write_times_ms_total
-#### HELP kubevirt_vmi_storage_write_times_ms_total Storage write operation time
 Storage write operation time
 
 ### kubevirt_vmi_storage_write_traffic_bytes_total
-#### HELP kubevirt_vmi_storage_write_traffic_bytes_total Storage write traffic in bytes
 Storage write traffic in bytes
 
-### kubevirt_vmi_network_traffic_bytes_total
-#### HELP kubevirt_vmi_network_traffic_bytes_total network traffic.
-Deprecated.
+### kubevirt_vmi_vcpu_seconds
+Amount of time spent in each state by each vcpu. Where `id` is the vcpu identifier and `state` can be one of the following: [`OFFLINE`, `RUNNING`, `BLOCKED`].
 
-## Virt Operator metrics
+### kubevirt_vmi_vcpu_wait_seconds
+Amount of time spent by each vcpu while waiting on I/O.
+
+### vmiEvictionBlockerDesc
+Indication for a VirtualMachine that its eviction strategy is set to Live Migration but is not migratable.
 
-### kubevirt_virt_controller_leading_total
-#### HELP kubevirt_virt_controller_leading Indication for an operating virt-controller.
+## Virt Operator Metrics
```

Comment:
I think we should drop this heading

Reply:
We have vmi metrics and virt operator metrics - do you want to mix them?

Reply:
I think its a problem to have this headings. Since we will also have other kubevirt operators here, like HCO/SSP/CDI metrics.